src/utils/wfs-utils.ts

```diff
@@ -0,0 +1,37 @@
+import { FeatureCollection } from 'geojson';
+
+import { WMSLayerProps } from '../config/types';
+
+interface propTypes {
+  lyr: WMSLayerProps;
+  startDate: any;
+  endDate: any;
+}
+
+export function fetchWMSLayerAsGeoJSON(
+  options: propTypes,
+): Promise<FeatureCollection> {
+  const { lyr, startDate, endDate } = options;
+
+  if (lyr.type !== 'wms') {
+    throw Error('unexpected layer type');
+  }
+
+  const wfsServerURL = `${lyr.baseUrl}/wfs`;
+
+  const params = new URLSearchParams();
```

Comment:
Let's mimick how we do things in `server-utils` a bit more. This function could probably live there as well.

Reply:
done with https://github.com/WFP-VAM/prism-frontend/pull/431/commits/844b6675ee6d13bff9080ede9232caf45a73f2c4

Reply:
resolving, but please reopen if there's an issue with it